### ISSUE TITLE
Question: Why not use the hidden _pair rule in the grammar.js?

### ISSUE THREAD

**Issue by v217** [ConradIrwin/tree-sitter-conl#1]
Hi!
A happy new year and another question:
I was wondering why you use `pair` instead of `_pair` in the file `grammar.js`? 
If you use `tree-sitter query` or `ast-grep` etc., you can access everything equally well, typing less and a less verbose syntax tree with the hidden `_pair` rule?
Thanks!

**Comment by ConradIrwin** [comment]
I didn’t think about it that hard, I think I looked at JSON for inspiration so maybe copying that.

would it be better for you the other way?

**Comment by v217** [comment]
Absolutely, pair is always followed by key. So the syntax tree is less verbose, while not loosing anything!

**Comment by v217** [comment]
The tree-sitter tests update all automatically, so if the three occurrences of `pair` are changed to `_pair` everything works as before.

**Comment by v217** [comment]
Maybe a problem:
This would cause backwards incompatibilities with other tools which query the tree-sitter ast.

**Comment by ConradIrwin** [comment]
As far as I know the only other thing using it is https://github.com/conradirwin/tree-sitter-conl, which I doubt uses it. Feel free to send a PR if you'd like

**Comment by v217** [comment]
Great! Thanks! In the first two closed PR I did forget to upload all the changed files, will try it again.

**Comment by v217** [comment]
Now I think it worked. Sorry for the delay, I am not too familiar with git.
